LWA/LWA_bifrost.py

```diff
@@ -684,6 +686,7 @@ def main(self):
             while not self.shutdown_event.is_set():
                 status = capture.recv()
         del capture
+        os.kill(os.getpid(), signal.SIGTERM)
```

Comment:
I don't know if this is actually needed here because of how the network capture works.  Maybe it would be if we had a way to exit this `while` loop based on the value of `status`.  Otherwise, the only way to exit this loop now is for a signal to have already been processed.